### ISSUE TITLE
Can't resolve library

### ISSUE THREAD

**Issue by nklugman** [ahmedrizwan/ReactiveAudioRecord#3]
Hey, I can't seem to get gradle to resolve the library. I'm pretty sure everything is set up correctly otherwise. Do you know what might be going on?

**Comment by ahmedrizwan** [comment]
Yeah I checked, you're right, it's not resolving... I think something's wrong with jCenter repo... I'll look into it... For the time being, you can add this under repositories and it should work
`maven {
            url  "http://dl.bintray.com/ahmedrizwan/maven"
 }`

**Comment by nklugman** [comment]
Thanks! Just FYI, I just checked again with the jcenter repo and still no luck.

**Comment by luongvo** [comment]
@ahmedrizwan You should consider to change to `jitpack.io` as a new way to host the library

**Comment by MuhamedFathy** [comment]
> Yeah I checked, you're right, it's not resolving... I think something's wrong with jCenter repo... I'll look into it... For the time being, you can add this under repositories and it should work
maven { url "http://dl.bintray.com/ahmedrizwan/maven" }

@ahmedrizwan Please update [README.md](https://github.com/ahmedrizwan/ReactiveAudioRecord/blob/master/README.md) with this line.
